EventFilter/CSCRawToDigi/interface/CSCTMBHeader.h

```diff
@@ -65,6 +75,23 @@ class CSCTMBHeader {
 
   void setNCFEBs(uint16_t ncfebs) { theHeaderFormat->setNCFEBs(ncfebs); }
 
+  /// == Run 3 CSC-GEM Trigger Format
+  uint16_t CLCT0_ComparatorCode() const { return theHeaderFormat->CLCT0_ComparatorCode(); }
```

Comment:
http://cms-sw.github.io/cms_coding_rules.html#2--naming-rules-1
2.8 Start method names with lowercase, use upper case initials for following words, e.g. collisionPoint().
Allowed exception: Implementation of virtual methods inherited from external packages, e.g. ProcessHits() method required by Geant4.

even though some of the old methods violate this rule, please conform to the style rule where possible